samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -181,6 +183,44 @@ The configuration of the DFU solution varies depending on the board target:
 | SUIT         | overwrite only mode            | * ``nrf54h20dk/nrf54h20/cpuapp``                                  |
 +--------------+--------------------------------+-------------------------------------------------------------------+
 
+Signature algorithm
+-------------------
+
+Each build target, that is supported by this sample, sets a specific signature algorithm for its DFU functionality.
+A signature algorithm generates a key pair that is used to sign the application image (the private key) and to verify the application image by the bootloader (the public key).
+The bootloader must have access to the public key for its image verification process.
+The choice of the signature algorithm and the implementation of the public key storage solution have an impact on the security properties of the overall DFU solution.
+
+The configuration of the signature algorithm and the public key storage solution in this sample varies depending on the board target:
+
++--------------------------------+-------------------------------------------------------------------+---------------------------+
+| Signature algorithm            | Board targets                                                     | Public key storage        |
++================================+===================================================================+===========================+
+| RSA-2048                       | * ``nrf52dk/nrf52832`` (only ``release`` configuration)           | Bootloader partition      |
+|                                | * ``nrf52833dk/nrf52833`` (only ``release`` configuration)        |                           |
+|                                | * ``nrf52840dk/nrf52840``                                         |                           |
+|                                | * ``nrf5340dk/nrf5340/cpuapp``                                    |                           |
+|                                | * ``nrf5340dk/nrf5340/cpuapp/ns``                                 |                           |
+|                                | * ``thingy53/nrf5340/cpuapp``                                     |                           |
+|                                | * ``thingy53/nrf5340/cpuapp/ns``                                  |                           |
++--------------------------------+-------------------------------------------------------------------+---------------------------+
+| ED25519                        | * ``nrf54l15dk/nrf54l05/cpuapp`` (only ``release`` configuration) | Key Management Unit (KMU) |
```

Comment:
`pure ED25519`

Reply:
pure / non-pure is the way in which the signature is calculated. The "pure" keyword does not carry any meaning in the cryptographic space, and we call it like this because we chose to name the corresponding Kconfig option - BOOT_SIGNATURE_TYPE_PURE - with the pure suffix.

I can add a column to the table to describe it

The algorithm is still called ED22519

Reply:
`pure / non-pure is the way in which the signature is calculated` -> so it adds some extra information (IIRC `pure` variant is safer too).

I would not introduce an additional column here. Maybe we could use e.g. `ED25519 (pure)` then

Reply:
Pure does not mean anything. How can the user infer from the "pure" keyword that the signature is calculated over the whole image? I will add another column